README.md

```diff
@@ -357,29 +357,34 @@ const secret = "hunter2";
 
 ```bash
 $ detect-secrets audit --help
```

Comment:
The audit help output has been updated to include the new report generation options.